app/src/main/res/layout-land/profile_chooser_profile_view.xml

```diff
@@ -37,19 +38,16 @@
 
       <com.google.android.material.imageview.ShapeableImageView
         android:id="@+id/profile_avatar_img"
-        android:layout_width="72dp"
-        android:layout_height="72dp"
+        style="@style/ShapeAbleImageView"
         android:layout_marginBottom="8dp"
-        app:shapeAppearanceOverlay="@style/ShapeAppearanceOverlay.RoundedShape"
-        app:strokeColor="@color/component_color_profile_chooser_activity_avatar_border_color"
-        app:strokeWidth="1dp"
-        profile:src="@{viewModel.profile.avatar}" />
```

Comment:
@Uticodes Why are these attributes removed? I these we need these to correct create profiles.

Otherwise even if user sets a profile picture it wont be visible. 

Create a new profile and make sure you add a profile picture and notice that it won't be visible on mobile-landscape profile chooser screen.

Reply:
Done